scripts/machine-launch-script.sh

```diff
@@ -1,82 +1,311 @@
+        CONDA_SUBCOMMAND=install
+        CONDA_ENV_BIN="${CONDA_INSTALL_PREFIX}/bin"
+    else
+        CONDA_ENV_BIN="${CONDA_INSTALL_PREFIX}/envs/${CONDA_ENV_NAME}/bin"
+        if [[ -d "${CONDA_INSTALL_PREFIX}/envs/${CONDA_ENV_NAME}" ]]; then
+            # 'create' clobbers the existing environment and doesn't leave a revision entry in
+            # `conda list --revivsions`, so use install instead
+            CONDA_SUBCOMMAND=install
+        else
+            CONDA_SUBCOMMAND=create
+        fi
+    fi
+
+    # to enable use of sudo and avoid modifying 'secure_path' in /etc/sudoers, we specify the full path to pip
+    CONDA_PIP_EXE="${CONDA_ENV_BIN}/pip"
+
+    # to enable use of sudo and avoid modifying 'secure_path' in /etc/sudoers, we specify the full path to conda
+    $SUDO "${CONDA_EXE}" "$CONDA_SUBCOMMAND" $DRY_RUN_OPTION -n "$CONDA_ENV_NAME" -y "${CONDA_PACKAGE_SPECS[@]}"
+
+
+    # Install python packages using pip that are not available from conda
+    #
+    # Installing things with pip is possible.  However, to get
+    # the most complete solution to all dependencies, you should
+    # prefer creating the environment with a single invocation of
+    # conda
+    PIP_PKGS=( \
+        fab-classic==1.19.1 \
+        mypy-boto3-ec2==1.21.9 \
+        sure==2.0.0 \
+    )
+    if [[ -n "$PIP_PKGS[*]" ]]; then
+        "${DRY_RUN_ECHO[@]}" $SUDO "${CONDA_PIP_EXE}" install "${PIP_PKGS[@]}"
+    fi
+
+
+    argcomplete_extra_args=()
+    if [[ "$INSTALL_TYPE" != system ]]; then
+        # if we're aren't installing into a system directory, then initialize argcomplete
+        # with --user so that it goes into the home directory
+        argcomplete_extra_args=( --user )
+    fi
+    "${DRY_RUN_ECHO[@]}" $SUDO "${CONDA_ENV_BIN}/activate-global-python-argcomplete" "${argcomplete_extra_args[@]}"
+
+
+    # temporarily hack the gcc specfile until
+    # https://github.com/conda-forge/ctng-compilers-feedstock/pull/91 is resolved upstream
+    gcc_machine=$($CONDA_ENV_BIN/gcc -dumpmachine)
+    gcc_version=$($CONDA_ENV_BIN/gcc --version | head -n1 | awk '{print $NF}')
+    CONDA_ENV_PREFIX=$(readlink -e $CONDA_ENV_BIN/..)
+    specdir=$CONDA_ENV_PREFIX/lib/gcc/$gcc_machine/$gcc_version
+
+    "${DRY_RUN_ECHO[@]}" $SUDO sed -i -e "/\*link_command:/,+1 s+%.*+& %{\!static:-rpath ${CONDA_ENV_PREFIX}/lib -rpath-link ${CONDA_ENV_PREFIX}/lib -disable-new-dtags} -L ${CONDA_ENV_PREFIX}/lib+" $specdir/specs
+    # use -idirafter to put the conda "system" includes where /usr/local/include would typically go
+    # in a system-packaged non-cross compiler
+    "${DRY_RUN_ECHO[@]}" $SUDO sed -i -e "/\*cpp_options:/,+1 s+%.*+& -idirafter ${CONDA_ENV_PREFIX}/include+" $specdir/specs
+    # cc1_options also get used for cc1plus... at least in 11.2.0
+    "${DRY_RUN_ECHO[@]}" $SUDO sed -i -e "/\*cc1_options:/,+1 s+%.*+& -idirafter ${CONDA_ENV_PREFIX}/include+" $specdir/specs
```

Comment:
I would ideally like to be able to remove this as well because it hacks the specfile installed by conda gcc.